src/com/magento/idea/magento2plugin/actions/generation/dialog/NewEntityDialog.java

```diff
@@ -107,7 +110,8 @@
         "PMD.ExcessiveImports",
         "PMD.GodClass",
         "PMD.TooManyMethods",
-        "PMD.CyclomaticComplexity"
+        "PMD.CyclomaticComplexity",
```

Comment:
Hi @eduard13 

Please consider that this part will be refactored in the scope of the separate task.

Reply:
Also, the extended testing will be performed in the scope of epic.